app/uk/gov/hmrc/tai/controllers/taxCodeChange/TaxCodeChangeController.scala

```diff
@@ -63,4 +64,19 @@ class TaxCodeChangeController @Inject()(authentication: AuthenticationPredicate,
         case ex: BadRequestException => BadRequest(Json.toJson(Map("reason" → ex.getMessage)))
       }
   }
+
+  def mostRecentTaxCodeRecords(nino: Nino, year: TaxYear): Action[AnyContent] = authentication.async {
+    implicit request =>
+
+      val latestTaxCodeRecords = taxCodeChangeService.latestTaxCodes(nino, year)
+
+      latestTaxCodeRecords.map { records =>
+        Ok(Json.toJson(ApiResponse(records, Seq.empty)))
```

Comment:
What about if `records` is empty? Should we return a `Not Found`? There is a similar response for a case like this in https://github.com/hmrc/tai/blob/master/app/uk/gov/hmrc/tai/controllers/JourneyCacheController.scala#L35